### DISCUSSION TITLE
Pressure gauge; Druck-Manometer

### DISCUSSION THREAD

**Discussion by Deinjo** [jomjol/AI-on-the-edge-device#3091] [General]
Hello everyone.

I am an enthusiastic home brewer. A pressure gauge is used for fermentation in bottles. By reading the pressure you can read the amount of carbon dioxide produced.

As I already use AI on the Edge for my gas and water meter, the idea came up that it could also be used to read the pressure gauge.

Reading analogue gauges is possible in principle. 
But if I understand correctly, this is only possible if it covers a range of 360° and the numbers go evenly from 0-9.

However, a pressure gauge has a different display. The range only covers approx. 270° and goes from 0..4 bar, for example.

Here is a picture:
<img width="385" alt="2024-06-01 14_23_00-Flaschenmanometer" src="https://github.com/jomjol/AI-on-the-edge-device/assets/48606253/0d9c6069-ec3d-4c3c-acf4-8634ae3291b7">

Is there perhaps a way to get the configuration right today? 
Or would it perhaps also be interesting to extend ‘AI on the Edge’ with something like this?

I could imagine that there could also be other use cases for this.

Greetings from Dortmund
Deinjo

**Comment by SybexX** [comment]
https://github.com/jomjol/AI-on-the-edge-device/issues/2954#issuecomment-2131458653
